frontend/packages/dev-console/integration-tests/features/pipelines/pipelines-Actions.feature

```diff
@@ -2,63 +2,63 @@ Feature: Perform the actions on Pipelines page
     As a user I want to view pipeline details, create, edit and delete the pipeline from Pipelines page
 
 Background:
-    Given user logged into the openshift application
-    And openshift cluster is installed with pipeline operator
-    And user is at the project namespace "AUT_MB_Demo" in dev perspecitve
+    Given openshift cluster is installed with pipeline operator
+    And user has selected namespace "aut-pipeline-actions"
 
 
 @regression, @smoke
-Scenario Outline: Newly created pipeline details in Pipelines page : P-03-TC01  
+Scenario Outline: Newly created pipeline details in Pipelines page : P-03-TC12 
    Given user is at pipelines page
+   And pipeline with name "<pipeline_name>" is present on Pipelines page
    When the user enters "<pipeline_name>" into the pipelines search bar
-   Then pipelines table displayed with column names "Name", "Namespace", "Last Run", "Task Status", "Last Run Status" and "Last Run Time"
+   Then pipelines table displayed with column names Name, Namespace, Last Run, Task Status, Last Run Status and Last Run Time
    And column Name display with value "<pipeline_name>"
-   And column Namespace display with value "AUT_MB_Demo"
+   # And column Namespace display with value "aut-pipeline-actions"
    And columns Last Run, Task Run Status, Last Run Status, Last Run Time with values display "-"
    And Create Pipeline button is enabled
    And kebab menu is displayed
 
 Examples:
 | pipeline_name |
-| nodejs-ex.git |
+| pipelines-aaa |
 
 
 @regression
-Scenario Outline: Kebab menu options of newly created pipeline in Pipelines page : P-04-TC09
+Scenario: Kebab menu options of newly created pipeline in Pipelines page : P-04-TC09
    Given user is at pipelines page
-   When the user enters "<pipeline_name>" into the pipelines search bar
-   And click kebab menu for the respective pipeline
-   Then kebab menu display with  options "Start", "Add Trigger", "Remove Trigger", "Edit Pipeline", "Delete Pipeline"
-
-Examples:
-| pipeline_name |
-| nodejs-ex.git |
+   When the user enters "nodejs-ex.git" into the pipelines search bar
+   And click kebab menu for the pipeline "nodejs-ex.git"
```

Comment:
```suggestion
   And user clicks kebab menu for the pipeline "nodejs-ex.git"
```

Reply:
user is not mandate for all steps, updated the word click to clicks